manim/animation/indication.py

```diff
@@ -113,33 +245,42 @@ def __init__(
         self.num_lines = num_lines
         self.flash_radius = flash_radius
         self.line_stroke_width = line_stroke_width
+        self.run_time = run_time
+        self.time_width = time_width
+        self.animation_config = kwargs
 
         self.lines = self.create_lines()
         animations = self.create_line_anims()
-        super().__init__(
-            *animations,
-            group=self.lines,
-            run_time=run_time,
-            **kwargs,
-        )
+        super().__init__(*animations, group=self.lines)
 
     def create_lines(self) -> VGroup:
+        # TODO change meaning of flash_radius
```

Comment:
**Question** whats this TODO ?

Reply:
A note to myself that I forgott to delete

Reply:
Removed :)